entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+until nc -w 1 -z db 5432; do
```

Comment:
This change will break the currently running instance on my server since it does not use the hostname `db` for PostgreSQL. This is suitable if you run the application using `compose` tool or if you manually map your hostname with container. In my case, the application and PostgreSQL are running on a separate network and their hosts were automatically taken from the container names by docker.

Reply:
Yep, I got it, then I think we could pass hostname in the command directive (docker-compose.yml). Something like this `command: ["./wait_for_psql_avail.sh", "db:5432"]` What do you think?

Reply:
Yes, it seems to me that this is a good solution. Only it seems to me that all such scripts for working with the docker should not be located in the root of the project or in the application folder. Maybe we should create a separate `docker` folder? Like for example done [here](https://github.com/wemake-services/wemake-django-template/tree/master/%7B%7Bcookiecutter.project_name%7D%7D).

Reply:
@nsidnev Yep, it's a good idea.